apis/nucleus/src/components/listbox/interactions/listbox-keyboard-navigation.js

```diff
@@ -191,6 +192,9 @@ export function getListboxInlineKeyboardNavigation({
   };
 
   const handleKeyDown = (event) => {
+    if (!keyboard.enabled) {
+      return undefined;
+    }
```

Comment:
I think we want to keep the cases that do updateKeyScrollOnHover(...) also when `!keyboard.enabled`